### PR TITLE
Add keyboard shortcuts to sources panel

### PR DESCRIPTION
**Pull request by gjmooney**

### CODE REVIEW

File: packages/schema/src/interfaces.ts [hunk 1]
```diff
@@ -219,6 +219,29 @@ export interface IJGISFormSchemaRegistry {
...
+  onSelect: ({ type, item, nodeId }: ILeftPanelClickHandlerParams) => void;
+}
+
+export interface ILeftPanelClickHandlerParams {
+  type: SelectionType;
+  item: string;
+  nodeId?: string;
+  event: MouseEvent;
+}
+
```
Comment (martinRenou) [inline packages/schema/src/interfaces.ts:244]: These types are specific to the left panel, should they not stay under `src/panelview/`? They are not used outside of this directory as far as I can see

**Review by martinRenou — CHANGES_REQUESTED** [review]
Thanks! I just have a small suggestion

File: packages/schema/src/interfaces.ts (hunk 1 above)
Comment (gjmooney) [reply packages/schema/src/interfaces.ts]: I moved the `IControlPanelModel` back to the `types` in `base` and the other two into the `leftPanel` file.

**Review by martinRenou — APPROVED** [review]
Thanks!

**Pull request merged**